Makefile.mac.mk

```diff
@@ -369,12 +386,31 @@ check-node-mac: build/frida_stripped-mac-$(build_arch)/lib/node_modules/frida ##
 	cd $< && $(NODE) --expose-gc node_modules/mocha/bin/_mocha
 
 
+install-mac: install-python-mac ##@utilities Install frida utilities (frida-{discover,ps,repl,trace})
+	@$(PYTHON) -measy_install colorama \
+		&& for b in "build/frida-mac-universal/bin"/*; do \
+			n=`basename $$b`; \
+			p="$(PREFIX)/bin/$$n"; \
+			grep -v 'sys.path.insert' "$$b" > "$$p"; \
+			chmod +x "$$p"; \
+		done
+
+uninstall-mac: ##@utilities Uninstall frida utilities
+	@for c in discover ps repl trace; do \
+		n=frida-"$$c"; \
+		if which "$$n" &> /dev/null; then \
+			p=`which "$$n"`; \
+			rm -f "$$p"; \
+		fi \
+	done
+
```

Comment:
Maybe an extra newline here to isolate this group? (Otherwise it looks connected to `.PHONY` below.)